### PR TITLE
Added rubocop-rake and rubocop-rspec support

### PR DESCRIPTION
**Pull request by l8556**
rename Rakefile

### CODE REVIEW

**Review by ShockwaveNN — CHANGES_REQUESTED** [review]
rubocop-rake and rubocop-rspec should be added to

1. Gemfile
2. .rubocop.yml

**Review by askonev — APPROVED** [review]

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -33,12 +33,51 @@ Naming/AccessorMethodName:
...
     - 'framework/palladium_helper.rb'
 
+# Offense count: 19
+# Configuration parameters: IgnoredMetadata.
+RSpec/DescribeClass:
+  Enabled: false
+
+# Offense count: 17
+# Cop supports --auto-correct.
+RSpec/EmptyLineAfterHook:
```
Comment (ShockwaveNN) [inline .rubocop_todo.yml:43]: I think this can be easily fixed

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -33,12 +33,51 @@ Naming/AccessorMethodName:
...
+  Enabled: false
+
+# Offense count: 37
+# Cop supports --auto-correct.
+# Configuration parameters: .
+# SupportedStyles: implicit, each, example
+RSpec/HookArgument:
```
Comment (ShockwaveNN) [inline .rubocop_todo.yml:50]: I think this probably can be fixed

File: .rubocop_todo.yml [hunk 3]
```diff
@@ -33,12 +33,51 @@ Naming/AccessorMethodName:
...
+  EnforcedStyle: each
+
+# Offense count: 2
+# Cop supports --auto-correct.
+RSpec/HooksBeforeExamples:
```
Comment (ShockwaveNN) [inline .rubocop_todo.yml:55]: I think this can be fixed

File: .rubocop_todo.yml [hunk 4]
```diff
@@ -33,12 +33,51 @@ Naming/AccessorMethodName:
...
+
+# Offense count: 14
+RSpec/MultipleExpectations:
+  Max: 3
+
+# Offense count: 23
+# Cop supports --auto-correct.
+# Configuration parameters: Strict, EnforcedStyle, AllowedExplicitMatchers.
+# SupportedStyles: inflected, explicit
+RSpec/PredicateMatcher:
```
Comment (ShockwaveNN) [inline .rubocop_todo.yml:73]: I think this can be fixed

File: .rubocop_todo.yml [hunk 5]
```diff
@@ -33,12 +33,51 @@ Naming/AccessorMethodName:
...
+  Enabled: false
+
 # Offense count: 1
-# Configuration parameters: ExpectMatchingDefinition, CheckDefinitionPathHierarchy, Regex, IgnoreExecutableScripts, AllowedAcronyms.
-# AllowedAcronyms: CLI, DSL, ACL, API, ASCII, CPU, CSS, DNS, EOF, GUID, HTML, HTTP, HTTPS, ID, IP, JSON, LHS, QPS, RAM, RHS, RPC, SLA, SMTP, SQL, SSH, TCP, TLS, TTL, UDP, UI, UID, UUID, URI, URL, UTF8, VM, XML, XMPP, XSRF, XSS
-Naming/FileName:
+# Cop supports --auto-correct.
+Rake/Desc:
```
Comment (ShockwaveNN) [inline .rubocop_todo.yml:78]: I think this can be fixed

**Review by ShockwaveNN — CHANGES_REQUESTED** [review]

File: spec/functional/documents/fodt_to_docx_functional_spec.rb [hunk 6]
```diff
@@ -1,23 +1,26 @@
 # frozen_string_literal: true
 
+# frozen_string_literal: true to all groups or subsets of all groups defined in the example of groups
```
Comment (ShockwaveNN) [inline spec/functional/documents/fodt_to_docx_functional_spec.rb:3]: Duplicate frozen_string_literal string

File: spec/smoke/convertion_tests_smoke_spec.rb [hunk 7]
```diff
@@ -13,27 +13,27 @@
       it "Check converting from #{format_from} to #{format}" do
         filepath = "#{StaticData::NEW_FILES_DIR}/new.#{format_from}"
         file_data = x2t.convert(filepath, format)
-        expect(File.exist?(file_data[:tmp_filename])).to be_truthy
+        expect(File).to exist(file_data[:tmp_filename])
       end
     end
   end
 
+  after do |example|
```
Comment (ShockwaveNN) [inline spec/smoke/convertion_tests_smoke_spec.rb:21]: I think this `after`  block should be places directly after `before` block

**Review by ShockwaveNN — CHANGES_REQUESTED** [review]

**Review by ShockwaveNN — APPROVED** [review]

**Pull request merged**